res/shaders/readme.md

```diff
@@ -19,6 +19,7 @@
 - `t14` is for ambient occlusion
 - `t15` is for the SkyBox resource
 - `t16` is for the Fog texture
+- `t17` is for deferred pass copy
```

Comment:
Will this be used for anything in the future PRs?

Reply:
It's already used in commit "Water: Add screen space refraction".

Reply:
Ah, I see